pyoos/parsers/ioos/get_observation.py

```diff
@@ -27,13 +29,13 @@ def __new__(cls, element):
 
         if version == "1.0":
             from pyoos.parsers.ioos.one.get_observation import GetObservation as GO10
-            return super(IoosGetObservation, cls).__new__(GO10, element=root)
+            return super(IoosGetObservation, cls).__new__(GO10)
```

Comment:
No idea to be honest. This is what was holding this PR for so long. I google and I found that this "works." Not sure if the tests are comprehensive enough to ensure this is correct though.